app.rb

```diff
@@ -88,7 +89,7 @@
 
 # Old doc pages are now at docs.everypolitician.org: redirect to them
 get '/about.html' do
-  docs_redirect('/', 'About')
+  redirect(settings.docs_url, '/', 'About')
```

Comment:
I think I'd leave a separate `docs_redirect` method that in turn calls the more general redirection method with a URI constructed with `settings.docs_url`, rather than having to change the existing callers.